content/en-us/reference/engine/classes/TextChatService.yaml

```diff
@@ -164,25 +182,46 @@ properties:
         <tbody>
           <tr>
             <td><b>RBXGeneral</b></td>
-            <td>For all messages.</td>
+            <td>For player messages.</td>
           </tr>
           <tr>
             <td><b>RBXSystem</b></td>
             <td>For system messages.</td>
           </tr>
           <tr>
             <td><b>RBXTeam[BrickColor]</b></td>
-            <td>For team messages. Name of the <code>Class.TextChannel</code> is the string <code>RBXTeam</code> followed by the name of <code>Class.Team.BrickColor</code>. For example, <code>RBXTeamCrimson</code> for a team with "Crimson" <code>Datatype.BrickColor</code>.</td>
+            <td>For team-specific player messages.
+            
+            Name of <code>Class.TextChannel</code> is <code>RBXTeam</code> followed by name of <code>Class.Team.TeamColor</code>. For example, <code>RBXTeamCrimson</code> for a team with "Crimson" <code>Datatype.BrickColor</code> as its <code>Class.Team.TeamColor</code> property.</td>
+          </tr>
+          <tr>
+            <td><b>RBXWhisper:[UserId1]_[UserId2]</b></td>
+            <td>For whisper messages between two players.
+            
+            The <code>Class.Player.UserId|UserIds</code> in <code>Class.TextChannel.Name</code> refer to players in the server who have a whisper channel. For example, <code>RBXWhisper:2276836_505306092</code> if players with <code>Player.UserId|UserIds</code> <code>2276836</code> and <code>505306092</code> began whispering.</td>
           </tr>
         </tbody>
       </table>
 
-      If false, `Class.Team`-based `Class.TextChannel|TextChannels` are **not**
-      automatically created and destroyed.
+      When a `Class.Team.TeamColor|TeamColor` is in use by any `Class.Team` within the `Class.Teams` service, a respective **RBXTeam[BrickColor]** `Class.TextChannel` is automatically created inside the **TextChannels** folder. When there are no remaining `Class.Team|Teams` with that `Class.Team.TeamColor`, the `Class.TextChannel` associated with the `Class.Team.TeamColor|TeamColor` is removed. This `Class.TextChannel` creates `Class.TextSource|TextSources` for all non-`Class.Player.Neutral|Neutral` players with the matching `Class.Team.TeamColor`, allowing them to use team-only chat if `Class.TextChatService.CreateDefaultCommands` is true.
```

Comment:
Why are these details being placed here instead of in the table layout above?

Reply:
It seemed like the table was given deliberately little information, so I only put in the table what was absolutely necessary to explain each channel type's purpose. Perhaps the table formatting doesn't look right on the final page when there's too much text on a row? It's hard to tell because I can't preview, but yes it would make more sense to include the extra details in the table.

Reply:
In my latest commit, I did my best to summarize all of the default channels inside the table, summarizing as best as possible without (perhaps) going into extreme levels of internal functionality. Let me know how this sounds and we can push this PR forward.